workflow/v1/README.md

```diff
@@ -0,0 +1,164 @@
+# Build Type: GitHub Actions Workflow
+
+This is a GitHub-maintained [SLSA Provenance](https://slsa.dev/provenance/v1)
+`buildType` that describes the execution of a GitHub Actions workflow.
+
+## Description
+
+```jsonc
+"buildType": "https://github.com/actions/buildtypes/workflow/v1"
+```
+
+This `buildType` describes the execution of a top-level [GitHub Actions]
+workflow that builds a software artifact.
+
+Only the following [event types] are supported:
+
+| Supported event type  | Event description                          |
+| --------------------- | ------------------------------------------ |
+| [`create`]            | Creation of a git tag or branch.           |
+| [`release`]           | Creation or update of a GitHub release.    |
+| [`push`]              | Creation or update of a git tag or branch. |
+| [`workflow_dispatch`] | Manual trigger of a workflow.              |
+
+[`create`]:
+  https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#create
+[`release`]:
+  https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#release
+[`push`]:
+  https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#push
+[`workflow_dispatch`]:
+  https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_dispatch
+
+This build type MUST NOT be used for any other event type unless this
+specification is first updated. This ensures that the [external parameters] are
+fully captured and that the semantics are unambiguous. Other event types are
+irrelevant for software builds (such as `issues`) or have complex semantics that
+may be error prone or require further analysis (such as `pull_request` or
+`repository_dispatch`). To add support for another event type, please open a
+[GitHub Issue].
+
+Note: Consumers SHOULD reject unrecognized external parameters, so new event
+types can be added without a major version bump as long as they do not change
+the semantics of existing external parameters.
+
+Note: This build type is **not** meant to describe execution of a subset of a
```

Comment:
This section should be removed right? As we do intend to use this build type when a reusable workflow is used. We should probably clarify this behaviour as the reusable workflow reference will be the `builder.id`.